src/pages/myAccount/MyProfile.tsx

```diff
@@ -32,8 +31,15 @@ const MyProfile = () => {
 
 const styles = {
     box: {
-        mt: 3,
-        mb: 5,
+        display: "flex",
+        flexDirection: "column",
+        justifyContent: "center",
+        alignItems: "center",
+        py: 4,
+        height: {
+            sm: "calc(100vh - 64px - 305px)", // 100vh - header height - footer height
```

Comment:
As above, hard coded values can cause problems if something changes. We should find a better, more dynamic, solution.